camunda-formio-tasklist-vue/src/components/layout/LeftSider.vue

```diff
@@ -246,10 +249,48 @@ export default class LeftSider extends Mixins(BaseMixin) {
     }
   }
 
-  beforeDestroy () {
+  beforeDestroy() {
     this.$root.$off("call-pagination");
     this.$root.$off("update-pagination-currentpage");
   }
 
 }
 </script>
+
+<style lang="scss" scoped>
+.task-list-container {
```

Comment:
Can you share screenshots of what this css change looks like?

Reply:
@krishnan-aot Nitheesh is trying to make this component generic to adopt the styles from the parent application. this will be published as new version and once it's ready to be integrated; we can try this out on service bc local/dev environments.